pkg/config/config.go

```diff
@@ -663,6 +692,39 @@ func DefaultConfig() *Config {
 			CommandTimeout:         "4h",
 			CommandTimeoutDuration: 4 * time.Hour,
 		},
+		DeleteOptimizations: DeleteOptimizations{
+			Enabled:          true,
+			Workers:          0, // auto-detect
+			BatchSize:        1000,
+			RetryAttempts:    3,
+			ErrorStrategy:    "continue",
+			FailureThreshold: 0.1,
+			CacheEnabled:     true,
+			CacheTTL:         30 * time.Minute,
+			S3Optimizations: struct {
```

Comment:
wrong definition here, use proper struct name instead of in place structure definition